examples/ner/run_ner.py

```diff
@@ -463,6 +469,22 @@ def main():
         "--do_lower_case", action="store_true", help="Set this flag if you are using an uncased model."
     )
 
+    parser.add_argument(
+        "--keep_accents", action="store_const", const=True, help="Set this flag if model is trained with accents."
+    )
+
+    parser.add_argument(
+        "--strip_accents", action="store_const", const=True, help="Set this flag if model is trained without accents."
+    )
+
+    parser.add_argument(
+        "--nouse_fast",
```

Comment:
We agreed to use positive form on cli argument if possible _(as it avoid double negation)_. Can you rename this parameter `--use-fast` and update the logic accordingly ?

It also make sense for the core library, as we set use_fast to False by default in 2.5.1 to reduce the breaking changes introduced with tokenizers.

Reply:
That makes sense. I'll fix it.